product_operating_unit/models/product_template.py

```diff
@@ -3,23 +3,29 @@
 # Copyright (C) 2019 Serpent Consulting Services
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 
-
-import logging
-
 from odoo import _, api, fields, models
-from odoo.exceptions import AccessError, RedirectWarning, ValidationError
-
-_logger = logging.getLogger(__name__)
+from odoo.exceptions import ValidationError
 
 
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
+    operating_unit_ids = fields.Many2many(
+        "operating.unit",
+        "product_operating_unit_rel",
+        string="Operating Units",
+        compute="_compute_operating_unit_ids",
+        store=True,
+        readonly=False,
+        default=lambda self: self._default_operating_unit_ids(),
```

Comment:
Do you still need the default, with the compute ?

Reply:
This was one of my thoughts, given the compute based on the categ_id
So piece of code is removed